content/issues/4/sonorous-medieval/style.css

```
@@ -84,15 +83,26 @@ cite.book::before {
   color: black;
 }
 
-/* figure 4 (provisional) */
+/* figure 4 */
+
+#fig4 .caption {
+  line-height: 1.25;
+}
 
 #fig4 table {
-  min-width: 50%;
+  width: 100%;
+  max-width: 32rem;
+  margin: 0 auto;
 }
 
 #fig4 table caption {
-  font-size: 24px;
+  font-size: 1rem;
   caption-side: top;
+  line-height: 1.25;
+
+  @media (min-width: 768px) {
+    font-size: 1.25rem;
```

I think setting a max-width on the caption fixes the table alignment on desktop.

```suggestion
    font-size: 1.25rem;
    max-width: 32rem;
```